code/obj/machinery/pipe/pipe_dispenser.dm

```diff
@@ -2,6 +2,16 @@
 TYPEINFO(/obj/machinery/disposal_pipedispenser)
 	mats = 16
 
+var/static/list/obj/machinery/disposal_pipedispenser/availdisposalpipes = list(
+	"Pipe" = 0,
+	"Bent Pipe" = 1,
+	"Junction" = 2,
+	"Y-Junction" = 4,
+	"Trunk" = 5,
```

Comment:
wher 3

Reply:
original didnt have it, guess ill add it